### ISSUE TITLE
Body data editor input error.

### ISSUE THREAD

**Issue by MokaCubrew** [mockoon/mockoon#1322]
**Describe the bug**

When entering json data into the body, it is entered in a different location from the cursor position. Editing data is too inconvenient in the editor.

[Mockoon.webm](https://github.com/mockoon/mockoon/assets/63692554/a2f708c4-125e-4904-b664-37407532ad4b)

**Mockoon version:**

6.2.0

**OS / OS version:**

Arch linux

**Comment by 255kb** [comment]
This is an odd bug that I wasn't able to reproduce.

However, I found various similar opened issues on the editor's repository and tried a fix on [a new branch](https://github.com/mockoon/mockoon/tree/bugfix/1322-ace-editor-cursor).
Could you build the app from this branch and check if it solves your issue? 

You will find the required steps in the [contributing guidelines](https://github.com/mockoon/mockoon/blob/main/CONTRIBUTING.md#build-and-run-the-applications-locally-during-development). You can try on a packaged application by running `npm run package:desktop:linux` as a last step.

**Comment by MokaCubrew** [comment]
Unfortunately, the same symptoms are still appearing. The further to the right I move, the greater the gap between the caret and the actual text input area becomes.

**Comment by 255kb** [comment]
I just noticed your editor font is not a monospaced one. I'm not sure it's something I can resolve on my side as the editor should be using one of these monospace fonts: `'Monaco', 'Menlo', 'Ubuntu Mono', 'Consolas', 'source-code-pro', monospace;`
And when digging deeper, I found many people having issues in arch Linux with font resolving, more precisely, `monospace` font resolving to Noto Sans, instead of Noto Sans Mono. 
[Google search on Arch's forum](https://www.google.com/search?q=arch+linux+noto+monospace+site:bbs.archlinux.org&sca_esv=1a0f9f064b6384e7&sca_upv=1&rlz=1C1CHBF_frLU877LU877&sxsrf=ACQVn097p0781GnEOZj-cGgSSscNZ16ENw:1713948765236&sa=X&ved=2ahUKEwibo8LavNqFAxX90AIHHW1JDosQrQIoBHoECBQQBQ&biw=2560&bih=1319&dpr=1)

Could there be a broken font configuration in your system? Thanks for checking!
